scripts/indy

```diff
@@ -0,0 +1,84 @@
+#! /usr/bin/env python3
```

Comment:
This file was deleted from `master` a long time ago; https://github.com/hyperledger/indy-node/commit/445452a4410f961c480a7344493932cc8090a221

I don't think it should be re-introduced.